app/routes/_gcn.user.email/email_announcements.server.ts

```diff
@@ -48,3 +54,66 @@ export async function deleteAnnouncementSubscription(
     email,
   })
 }
+
+export async function sendNewsAnnouncementEmail(
```

Comment:
The other functions are of the form verbAnnouncementObject, not verbNewsAnnouncementObject.

```suggestion
export async function sendAnnouncementEmail(
```